CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 
 ## Unreleased: mitmproxy next
 
+* mitmweb: make clicking suggestions below `Intercept` input change intercept
+  ([#8026](https://github.com/mitmproxy/mitmproxy/pull/8026), @xBZZZZ)
```

Comment:
I would change that to: `mitmweb: show intercept filter tag at the bottom for default options`